src/help/guides/getting-started.md

```diff
@@ -63,42 +64,42 @@ This requires iOS 13.3+ or iPadOS 13.4+, the app will CRASH otherwise. Paperback
    <el-tag type="warning">Not the Windows Store app!</el-tag>
 1. Download and install [iCloud for Windows 8](https://support.apple.com/en-us/HT204283).
    <el-tag type="warning">Not the Windows Store app!</el-tag>
-   > Can't find the link to download **iCloud** for Windows 8? [Image](https://imgur.com/a/P1ef4Wd)
-1. Restart your PC after installing both **iTunes** and **iCloud**.
+  > Can't find the link to download **iCloud** for Windows 8? [GIF](https://imgur.com/a/P1ef4Wd)
```

Comment:
The citation need to be indented to prevent the numbering from being reset on the next step.
```suggestion
	> Can't find the link to download **iCloud** for Windows 8? [GIF](https://imgur.com/a/P1ef4Wd)
```